test/Microsoft.AspNetCore.Routing.Tests/DefaultLinkGeneratorTest.cs

```diff
@@ -67,7 +67,7 @@ public void GetLink_Fail_ThrowsException()
         public void TryGetLink_Fail()
         {
             // Arrange
-            var endpoint = CreateEndpoint("{controller}/{action}");
+            var endpoint = EndpointHelper.CreateMatcherEndpoint("{controller}/{action}");
```

Comment:
Suggestion. Make this a `using static`. Seeing `Helper` all over the code just makes my eyes bleed.